docs/balance-checker.html

```diff
@@ -0,0 +1,320 @@
+                   meta: dumMeta(chainId)
+                 }, host(chainId))
+                 const {pred, keys} = response.result.data
+                 guardElem.textContent = pred + "\r\n" + keys.reduce((accum, key) => JSON.stringify(key)+"\r\n" + accum, "")
+                 guardElem.className = "";
+             } catch {
+               guardElem.textContent = JSON.stringify(result.data.guard)
+               guardElem.className = "";
+             }
+           } else {
+             guardElem.textContent = JSON.stringify(result.data.guard)
+             guardElem.className = "";
+           }
+         }
+        } else if (result.status==="failure" && result.error.message.slice(0,24)==="with-read: row not found"){
+          balanceElem.textContent = "N/A"
+          balanceElem.className = "warning";
+          guardElem.textContent = "Doesn't Exist"
+          guardElem.className = "warning";
+        } else {
+          balanceElem.textContent = "N/A"
+          balanceElem.className = "error";
+          guardElem.textContent = "Request Failed"
+          guardElem.className = "error";
+        }
+      } catch(e){
+        console.log(e)
+        document.getElementById(`chain-${chainId}-balance-data`).textContent = "N/A"
+        document.getElementById(`chain-${chainId}-balance-data`).className = "error";
+        document.getElementById(`chain-${chainId}-guard-data`).textContent = "Request Failed"
+        document.getElementById(`chain-${chainId}-guard-data`).className = "error";
+      }
+    }
+
+    window.addEventListener('load', function (event) {
+      if (localStorage.getItem("version")==="1" && localStorage.getItem("kadena-server")){
+         document.getElementById("server").value = localStorage.getItem("kadena-server");
+      } else {
+         document.getElementById("server").value = "api.chainweb.com"
+         localStorage.setItem("version","1")
+      }
+      if (localStorage.getItem("kadena-token")){
+        document.getElementById("token").value = localStorage.getItem("kadena-token");
+      }
+      if (localStorage.getItem("kadena-account")){
+        document.getElementById("account").value = localStorage.getItem("kadena-account");
+      }
+    }, false);
+
+    document.addEventListener('click', async function (event) {
+      chainBal = {}
+      if (!event.target.matches('#balance')) return;
+      event.preventDefault();
+      const server = document.getElementById('server').value;
+      const token = document.getElementById('token').value;
+      const info = await getVersion(server);
+      const host = (chainId) => `https://${server}/chainweb/0.0/${info.nv}/chain/${chainId}/pact`;
+      if (document.getElementById('account').value.length<3 || document.getElementById('account').value.length>256){
+        document.getElementById("account-field").className = "field error"
+        document.getElementById("kadena-form").className = "ui form error"
```

Comment:
This causes the entire form to disappear, when it should render the error message.